source/ways-of-working/standards/flyway-database-migrations.html.md.erb

```diff
@@ -0,0 +1,126 @@
+---
+title: Flyway database migration
+last_reviewed_on: 2022-03-17
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+## How to apply Flyway migration as part of pipeline
```

Comment:
should have a little background before this heading on why, probably linking to the other page.

Maybe some words around preventing application startup failure for larger migrations due to Kubernetes probes it's essential that databases are migrated outside of Kubernetes via the Continuous delivery pipeline?